strongbox-data-service/src/main/java/org/carlspring/strongbox/config/HazelcastConfiguration.java

```diff
@@ -18,13 +21,24 @@
 public class HazelcastConfiguration
 {
 
+    public static final int ARTIFACT_ENTRY_CACHE_INVALIDATE_INTERVAL = 60;
+    
+    public static MapConfig nearCacheConfig(String name)
+    {
+        return new MapConfig().setName(name).setNearCacheConfig(new NearCacheConfig().setCacheLocalEntries(true)
```

Comment:
Just for my better understanding, what is this near cache config used for?

Reply:
`NearCache` is fast cache which is not shared between nodes. It used to speed up `ArtifactEntry` fetches within HTTP request, if we fetch it more then once within current request.

Reply:
Thanks for the explanation! :)